lib/nsq/connection.rb

```diff
@@ -32,6 +32,7 @@ def initialize(opts = {})
       @msg_timeout = opts[:msg_timeout] || 60_000 # 60s
       @max_in_flight = opts[:max_in_flight] || 1
       @ssl_context = opts[:ssl_context]
+      @tls_v1 = !!opts[:tls_v1]
       validate_ssl_context! if @ssl_context
```

Comment:
@alieander Resolved in https://github.com/wistia/nsq-ruby/pull/26/commits/5caa7bd53dfea57db6ab86f769c46db807de7deb